assests/javaScript/home.js

```diff
@@ -0,0 +1,62 @@
+const movieList = document.getElementById("movie-list");
+const apiKey = "d2aa1d6a7ef9316cd7e2270dd937b843";
+
+fetch(
+  `https://api.themoviedb.org/3/movie/top_rated?api_key=${apiKey}&language=en-US&page=1`
+)
+  .then((res) => res.json())
+  .then((data) => showMovies(data.results))
+  .catch((error) => alert(`error: ${error}`));
```

Comment:
Just as a friendly reminder; I know you are going to remove the alert.

Reply:
yessss thanks